flexget/plugins/input/anilist.py

```diff
@@ -67,12 +68,12 @@ class AniList(object):
 
     @cached('anilist', persist='2 hours')
     def on_task_input(self, task, config):
-        entries = []
         if isinstance(config, str):
             config = {'username': config}
         selected_list_status = config['status'] if 'status' in config else ['current', 'planning']
         selected_release_status = config['release_status'] if 'release_status' in config else ['all']
         selected_formats = config['format'] if 'format' in config else ['all']
+        lightweight = config['title_only'] if 'title_only' in config else False
```

Comment:
This sort of statement is more easily written using `dict.get`
```suggestion
        lightweight = config.get('title_only', False)
```